### PR TITLE
Leaderboard: setup project

### PR DESCRIPTION
**Pull request by Ghilain**
Requesting for code review for this project
Project requirements:

- IMPORTANT NOTE This is the first time you will use [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md). Keep that in mind!

- Create an npm project with webpack. (If you need a refresher, check the previous project or webpack exercise).

- Write plain HTML markup with minimum styling (just to make the layout), because you will implement styles in the following steps.
- Don't make API calls (you'll do it in a following lesson).
- Use ES6 modules, with import and export.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    "hint": "^6.2.0",
+    "html-webpack-plugin": "^5.5.0",
+    "sass-loader": "^12.6.0",
+    "style-loader": "^3.3.1",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0",
+    "webpack-dev-server": "^4.9.0"
+  }
```
Comment (omar-muhamad) [inline package.json]: - [x] Great job so far but kindly you need to include `webpack`, `webpack-cli`, and `webpack-dev-server` as `devDependencies` too.
You can just do that by running those two command lines `npm install --save-dev webpack` and `npm install --save-dev webpack-cli`

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @Ghilain,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [x] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: package.json (hunk 1 above)
Comment (Ghilain) [reply package.json]: Thank you @Omar-Muhamad for support

Comment (omar-muhamad) [reply package.json]: You're welcome ☺️

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @Ghilain,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**